### ISSUE TITLE
Concurrent versions problem

### ISSUE THREAD

**Issue by sergefedorow** [ONLYOFFICE/onlyoffice-alfresco#13]
Critical Problem!
Blocked documents have "Edit in ONLYOFFICE" item in menu. So one user can change document, during another user block it.
Original parashift connector hide this menu item on blocked documents.

**Comment by sergefedorow** [comment]
Andrey, fix #13 hide 'Edit in ONLYOFFICE' for all blocked documents, including onlyoffice. Collaboration is gone. It mast hide  'Edit in ONLYOFFICE' for all documents blocked by any way, excluding onlyoffice.

**Comment by Eloncase** [comment]
Just tested it after bc7c5841e1087a32f3cfc71dcb0cb8a2c3a1f985 with two different users on a file inside `Shared Files` folder and everything works just fine. Have you built the plugin from `develop` or `bugfix/editing-session-hash` branch?

**Comment by sergefedorow** [comment]
Sorry, I rebuild project from bugfix/editing-session-hash, including fix #14 #15 #12 #13, and it works properly. Thanks!

**Comment by sergefedorow** [comment]
Menu has changed, but the bug still present in another form. Then one user block document for editing and other user has opened old, not reloaded page, his have menu item "Edit in ONLYOFFICE" and can make fork editing without any warning.
MS Word, for example, check document for blocks during opening and give the user warning message.

**Comment by sergefedorow** [comment]
Than one user block document for editing and other user has old, not reloaded page, his have menu item "Edit in ONLYOFFICE". Press this button give onlyoffice-edit.get.js error 500. It would be nice give user warning message, some thing like MS Word do. Word check document for blocks during opening and give the user clear warning message.

**Comment by sergefedorow** [comment]
Than one user block document for editing and other user has old, not reloaded page, his have menu item "Edit in ONLYOFFICE". Press this button give onlyoffice-edit.get.js error 500. It would be nice give user warning message, some thing like MS Word do. Word check document for blocks during opening and give the user clear warning message.

**Comment by LinneyS** [comment]
Hi @sergefedorow 
Can you open a new Issue for this?
